src/map/map.c

```diff
@@ -5275,12 +5275,13 @@ void read_map_zone_db(void) {
 		ShowStatus("Done reading '"CL_WHITE"%d"CL_RESET"' zones in '"CL_WHITE"%s"CL_RESET"'.\n", zone_count, config_filename);
 		
 		/* post-load processing */
+		map->zone_all.merge_type = MZMT_NEVERMERGE;
```

Comment:
I only see `MZMT_NEVERMERGE` and `MAMT_MERGEABLE` being used. If this relies zeroed/allocated zones having an implicit `MZMT_NORMAL` (0), I believe it'd be better to explicitly initialize those explicitly.